README.md

```diff
@@ -1,806 +1,10 @@
-# App Functions SDK (Golang) - Beta
 
-Welcome the App Functions SDK for EdgeX. This sdk is meant to provide all the plumbing necessary for developers to get started in processing/transforming/exporting data out of EdgeX. 
+# Application Functions SDK
 
-Table of contents
-=================
+For documentation please visit https://docs.edgexfoundry.org/microservices/application/ApplServices/.
+For a list of examples please visit https://docs.edgexfoundry.org/examples/AppServiceExamples/
```

Comment:
This sections doesn't exists, so just link to the holding repo?
`https://github.com/edgexfoundry-holding/app-service-examples`